resources/public/api/conf/2.0/scenarios/uk_property_non_fhl_period_summary_create.md

```diff
@@ -0,0 +1,39 @@
+<p>Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.</p>
```

Comment:
The wording in here is correct but there already exists a historic_uk_period_summary_create_scenarios.md file where the wording is wrong. 

- Please move your content to that file.
- Then (optionally) you can remove this file and reference that file from the RAML (the non-fhl and fhl scenarios are the same).